README.md

```diff
@@ -189,3 +189,16 @@ To generate the `standalone.py` script, run the following command ([ruff](https:
 ```bash
 make standalone
 ```
+
+## Developer setup
+
+To collaborate on this repository, please follow these steps:
+
+1. Install [uv](https://docs.astral.sh/uv/getting-started/installation/)
+2. Run following commands to prepare your local environment
+
+```bash
+uv sync
+uv venv
```

Comment:
I've found out this command is actually not necessary. `uv sync` by itself creates a `.venv` if it's not already present. That's neat. 🙂